docs/source/Advanced-Usage.md

```diff
@@ -202,6 +203,17 @@ You should follow the [official guide](https://cloud.google.com/web-risk/docs/qu
 Then you can copy the generated JSON key file in the directory `configuration` of the project and change its name to `service_account_keyfile.json`.
 This is the default configuration. If you want to customize the name or the location of the file, you can change the environment variable `GOOGLE_APPLICATION_CREDENTIALS` in the `env_file_app` file.
 
+## Customization options for connectors
+Connectors by nature are designed to run independently after each analysis. There is no option to dynamically supply a `runtime_configuration` like analyzers, however you can always change the `configuration/connectors_config.json` as you wish with the following options:
+* `MISP`:
+    * `ssl_check`: (default True), enable SSL certificate server verification. Change this if your MISP instance has not SSL enabled.
+    * `debug`: (default False) enable debug logs
+    * `tlp` (Traffic Light Protocol): (default white) change this as per your organization's threat sharing conventions.
+* `OpenCTI`:
+    * `ssl_verify`: (default `true`), enable SSL certificate server verification. Change this if your OpenCTI instance has not SSL enabled.
```

Comment:
The "default True" and "default `true`" strings mean the same thing but can be confusing to the users. Just write "default `true`" or "default `false`" everywhere because JavaScript/JSON.